src/watertap_contrib/seto/unit_models/surrogate/vagmd_surrogate.py

```diff
@@ -0,0 +1,720 @@
+
+        # sf = iscale.get_scaling_factor(
+        #     self.avg_feed_props[0].mass_frac_phase_comp["Liq", "TDS"]
+        # )
+        iscale.constraint_scaling_transform(self.eq_avg_salinity_feed_tank, 1e-1)
+
+        # sf = iscale.get_scaling_factor(
+        #     self.avg_feed_props[0].flow_vol_phase["Liq"]
+        # )
+        iscale.constraint_scaling_transform(self.eq_feed_volumetric_flow_rate, 1e5)
+
+        sf = iscale.get_scaling_factor(self.permeate_flux)
+        iscale.constraint_scaling_transform(self.eq_permeate_flux, sf)
+
+        # sf = iscale.get_scaling_factor(
+        #     self.evaporator_out_props[0].temperature
+        # )
+        iscale.constraint_scaling_transform(self.eq_evaporator_outlet_temp, 1e-2)
+
+        # sf = iscale.get_scaling_factor(
+        #     self.condenser_out_props[0].temperature
+        # )
+        iscale.constraint_scaling_transform(self.eq_condenser_outlet_temp, 1e-2)
+
+        # sf = iscale.get_scaling_factor(
+        #     self.permeate_props[0].flow_vol_phase["Liq"]
+        # )
+        iscale.constraint_scaling_transform(self.eq_permeate_volumetric_flow_rate, 1e5)
+
+        sf = iscale.get_scaling_factor(self.log_mean_temp_dif)
+        iscale.constraint_scaling_transform(self.eq_log_mean_temp_dif, sf)
+
+        sf = iscale.get_scaling_factor(self.thermal_power)
+        iscale.constraint_scaling_transform(self.eq_thermal_power, sf)
+
+        sf = iscale.get_scaling_factor(self.thermal_energy)
+        iscale.constraint_scaling_transform(self.eq_thermal_energy, sf)
+
+    # def initialize_build(self, state_args=None,
+    #                solver=None, optarg=None, outlvl=idaeslog.NOTSET):
+    #     init_log = idaeslog.getInitLogger(self.name, outlvl, tag="unit")
+    #     init_log.info_low("Starting initialization...")
+
+    """
+    Equation to calculate pressure drop
+    """
+
+    def _get_pressure_drop(self, flow_rate, salinity):
+        if self.config.module_type == "AS7C1.5L":
+            coefficients = [
+                -158.2007422,
+                0.39402609,
+                0,
+                0.000585345,
+                8.93618e-5,
+                -0.000287828,
+            ]
+        else:  # self.config.module_type == 'AS26C7.2L'
+            coefficients = [
+                -72.53793298,
+                0.110437201,
+                0,
+                0.000643495,
+                0.000189924,
+                -0.001111447,
+            ]
+
+        return (
+            coefficients[0]
+            + coefficients[1] * flow_rate
+            + coefficients[2] * salinity
+            + coefficients[3] * flow_rate * salinity
+            + coefficients[4] * flow_rate**2
+            + coefficients[5] * salinity**2
+        )
+
+    """
+    Surrogate equations for membrane performance
+    """
+
+    def _get_membrane_performance(self, TEI, FFR, TCI, SgL):
+        # Model parameters
```

Comment:
These lists should have at least some minimal description. Pflux may be more obvious, but "Coder" is not.

Reply:
Variable name updated.